compose/kafka-compose.yaml

```diff
@@ -63,20 +63,56 @@ services:
     image: 'quay.io/kroxylicious/proxy:0.18.0-SNAPSHOT'
     container_name: kroxylicious
     command: --config /opt/kroxylicious/config/proxy-config.yaml
+    environment:
+      JAVA_ENABLE_DEBUG: true
+      JAVA_DEBUG_SUSPEND: true
```

Comment:
this needs to be false to remain, but I think its a good idea to show it as available.